views/task-items.html

```diff
@@ -9,28 +10,32 @@
                 {{item.task.creationDate | datetime }}
                 <b>Assigned to:</b>
                 <input name="assignee" ng-show="item.access.edit" class='assign inline-edit' placeholder="Assign to"
-                       ng-model="item.task.assignee" required>
+                       ng-model="item.task.assignee" required user-exist>
+                <span class="error">
+                    User does not exist
+                </span>
                 <input type="submit" class="btn assign-btn" ng-click="reassignTask(item)" value="Assign"
-                       ng-show="item.access.edit" ng-disabled="taskItemForm.assignee.$invalid">
+                       ng-show="item.access.edit">
             </div>
 
-            <textarea required name="description" class="inline-edit" rows=1
+            <textarea required name="description" class="inline-edit" rows=1 auto-rows
                       ng-model="item.task.description"
+                      ng-trim="false"
                       ng-focus="onFocus(item);"
                       ng-readonly="!item.access.edit"></textarea>
 
-            <input type="submit" class="btn save-btn" ng-click="saveDescription(item); item.focused = false;"
-                   value="Save"
-                   ng-disabled="taskItemForm.description.$invalid" ng-show="item.access.edit && item.focused">
             <input type="submit" class="btn cancel-btn" ng-click="restoreDescription(item); item.focused = false;"
                    value="Cancel"
                    ng-show="item.access.edit && item.focused">
+            <input type="submit" class="btn save-btn" ng-click="saveDescription(item); item.focused = false;"
+                   value="Save"
+                   ng-show="item.access.edit && item.focused">
```

Comment:
My model (I mean "Tasks") actually doesn't know about things like "hovered" and "focused". It returns only tasks and accesses.
And the "widgetController" appends this attributes to the data returned by model.
I want fully understand why in this case it's bad and after that make a changes.